Mobile Buy SDK/Mobile Buy SDK/Models/Mobile Buy SDK.xcdatamodeld/Mobile Buy SDK.xcdatamodel/contents

```diff
@@ -321,7 +321,7 @@
         </userInfo>
     </entity>
     <entity name="Collection" representedClassName="BUYCollection" syncable="YES">
-        <attribute name="createdAt" optional="YES" attributeType="Date" syncable="YES"/>
+        <attribute name="createdAt" optional="YES" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
```

Comment:
The short answer is "No", since I had to change the entity to add the property. Some of them are automated. Which ones are you particularly concerned about? All the changes to `_BUYPRoduct.[hm]` are automated.